### PR TITLE
User creation

### PR DESCRIPTION
**Pull request by JoshuaLawson8**
Through page for user creation

### CODE REVIEW

**Review by charlotte-zhuang — APPROVED** [review]
I think this should be ok, as long as the change I made to `Chat.jsx` doesn't break anything.

File: src/components/Chat.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const ChatApp = (props) => {
         <div className={styles.App}>
           <SendBirdApp
             appId={process.env.REACT_APP_SENDBIRD_ID}
-            userId={props.user.email}
+            userId={props.user.userId}
```
Comment (charlotte-zhuang) [inline src/components/Chat.jsx]: @euniceyoh Is this change ok? Since `props.user.email` might be incorrect unless `props.user.emailVerified` is true.

Comment (euniceyoh) [reply src/components/Chat.jsx]: I changed sendbird's user's id to a user's name, even though they're not guaranteed to be unique, bc i needed to get the PT's email through marketplace, and it wasn't part of the schema.

**Review by isabellapiz — APPROVED** [review]
great job!

**Review by christinang3030 — APPROVED** [review]
Cool stuff

**Pull request merged**